src/web/app/pages-admin/admin-search-page/admin-search-page.component.html

```diff
@@ -5,9 +5,9 @@
     </p>
 
     <div class="input-group">
-      <input type="text" class="form-control" [(ngModel)]="searchQuery" (keyup.enter)="search()">
+      <input type="text" class="form-control" id="filter-query" [(ngModel)]="searchQuery" (keyup.enter)="search()">
```

Comment:
I don't think it is a good practice to give id for element just for the sake of e2e testing? Asking the e2e testing team about how they do it?

Reply:
I asked Jacob. He said that it was okay. Existing tests do the same